src/__tests__/vdxf/appencryptionrequestdetails.test.ts

```diff
@@ -0,0 +1,80 @@
+import { BN } from "bn.js";
+import { AppEncryptionRequestDetails, AppEncryptionRequestDetailsJson, CompactIdAddressObject } from "../../vdxf/classes";
+import { TransferDestination } from "../../pbaas/TransferDestination";
+import { BigNumber } from "../../utils/types/BigNumber";
+
+// Helper function to create TransferDestination from address string
+function createCompactIdAddressObject(flags: BigNumber, address: string): CompactIdAddressObject {
```

Comment:
I added fromIAddress and toIAddress functions to CompactIdAddressObject, you can add tests for those here instead of this helper function I think